src/Trigger.tsx

```diff
@@ -1,6 +1,10 @@
-import React from 'react'
+import React, {FunctionComponent} from 'react'
 
-function Trigger({ onTrigger }) {
+type TriggerProps = {
+    onTrigger : () => void
```

Comment:
I think we should add prettier since otherwise the styling is a bit off. E.g. there should be a trailing semicolon here and non-standard code style can be a little distracting